### ISSUE TITLE
Typo *can verified*

### ISSUE THREAD

**Issue by Benjamin-Loison** [Benjamin-Loison/opentimestamps-client#12]
See [blob/185a5997cf315402747223a1feae18e669ae08f2/doc/git-integration.md?plain=1#L175-L176](https://github.com/opentimestamps/opentimestamps-client/blob/185a5997cf315402747223a1feae18e669ae08f2/doc/git-integration.md?plain=1#L175-L176).

**Comment by petertodd** [comment]
Did you mean to open these issues at https://github.com/opentimestamps/opentimestamps-client/?

**Comment by Benjamin-Loison** [comment]
Hey @petertodd, glad to see you here :smile:

TLDR: If it makes sense at some point I would open these issues at https://github.com/opentimestamps/opentimestamps-client but first I let my issues mature here on my fork.

I have an unconventional, it seems, way of experiencing softwares.
Most of time I just notice something about a software (a typo, an enhancement, an error...) not blocking me and do not have much time to investigate it, so I fork the software and open a public issue with the details I have on the moment and let it as is. If the issue happens again at another time, then I come back to the issue and write new details depending on the new situation I am in if it is different, otherwise I investigate a bit the issue. If at some point I consider the issue mature enough, like I am not going to look like a noob, by having looked upstream issues/pull requests for a similar one, checked `{README,CONTRIBUTING}.md` and found a workaround or a solution if one looks like a low hanging fruit, then I open an issue upstream.

For referencing: I already detailed similarly my methodological approach on [Codeberg Matrix](https://matrix.to/#/!wkyjxWeAOBpKMvHbno:matrix.org/$RHZM-E8FsCAQPC8Sar9kDTYjvRUoiFcnaKEezO7TFgI), see [Benjamin_Loison/codeberg/issues/1](https://gitea.lemnoslife.com/Benjamin_Loison/codeberg/issues/1).

You fixed the issue upstream, with [opentimestamps-client/commit/4f37bb9e9f8ca56099af9c0f290fa77e0f974fbd](https://github.com/opentimestamps/opentimestamps-client/commit/4f37bb9e9f8ca56099af9c0f290fa77e0f974fbd), thank you!

I am curious, how have you figured out my issues?
